tests/test_losses.py

```diff
@@ -78,25 +80,27 @@ def test_categorical_likelihood(make_data):
 
 def test_sum_likelihood():
```

Comment:
These tests don't appear to check the error case where broadcasting occurs?

Reply:
Good point!

Reply:
That would be good, thanks man :-)